0110/streams/index.html

```diff
@@ -1,275 +1,311 @@
 <!--
- Licensed to the Apache Software Foundation (ASF) under one or more
- contributor license agreements.  See the NOTICE file distributed with
- this work for additional information regarding copyright ownership.
- The ASF licenses this file to You under the Apache License, Version 2.0
- (the "License"); you may not use this file except in compliance with
- the License.  You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
- Unless required by applicable law or agreed to in writing, software
- distributed under the License is distributed on an "AS IS" BASIS,
- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- See the License for the specific language governing permissions and
- limitations under the License.
--->
-
-<script><!--#include virtual="../js/templateData.js" --></script>
-
+   Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
I need to point out that if we made a new release from 0.11.0 and do the copy-paste from `kafka` repo these changes will be auto-reverted as well. Do you want to submit a corresponding PR for `kafka` against 0.11.0 branch to keep the docs in-sync?